/home/airsim/gym-airsim/environments.py:273: UserWarning: 'gym-eolic-position-aereo-rgb' is not a legal ROS base name. This may cause problems with other ROS tools.
  rospy.init_node(node)
Traceback (most recent call last):
  File "exec.py", line 52, in <module>
    observation = env.reset()
  File "/home/airsim/.local/lib/python3.8/site-packages/gymnasium/wrappers/order_enforcing.py", line 61, in reset
    return self.env.reset(**kwargs)
  File "/home/airsim/.local/lib/python3.8/site-packages/gymnasium/wrappers/env_checker.py", line 57, in reset
    return env_reset_passive_checker(self.env, **kwargs)
  File "/home/airsim/.local/lib/python3.8/site-packages/gymnasium/utils/passive_env_checker.py", line 186, in env_reset_passive_checker
    result = env.reset(**kwargs)
  File "/home/airsim/gym-airsim/environments.py", line 453, in reset
    self._start_pose()
  File "/home/airsim/gym-airsim/environments.py", line 300, in _start_pose
    self.vehicle.go_home()
  File "/home/airsim/gym-airsim/airsim-helper/ros_helper.py", line 498, in go_home
    self.vehicle.set_pose([0,0,0], [0,0,0])
AttributeError: 'ActPosition' object has no attribute 'vehicle'
Traceback (most recent call last):
  File "exec.py", line 52, in <module>
    observation = env.reset()
  File "/home/airsim/.local/lib/python3.8/site-packages/gymnasium/wrappers/order_enforcing.py", line 61, in reset
    return self.env.reset(**kwargs)
  File "/home/airsim/.local/lib/python3.8/site-packages/gymnasium/wrappers/env_checker.py", line 57, in reset
    return env_reset_passive_checker(self.env, **kwargs)
  File "/home/airsim/.local/lib/python3.8/site-packages/gymnasium/utils/passive_env_checker.py", line 186, in env_reset_passive_checker
    result = env.reset(**kwargs)
  File "/home/airsim/gym-airsim/environments.py", line 453, in reset
    self._start_pose()
  File "/home/airsim/gym-airsim/environments.py", line 300, in _start_pose
    self.vehicle.go_home()
  File "/home/airsim/gym-airsim/airsim-helper/ros_helper.py", line 498, in go_home
    self.vehicle.set_pose([0,0,0], [0,0,0])
AttributeError: 'ActPosition' object has no attribute 'vehicle'
Connected!
Client Ver:1 (Min Req: 1), Server Ver:1 (Min Req: 1)